src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -60,6 +60,14 @@ using std::auto_ptr;
 
 /* ****************************************************************************
 *
+* OtisTreatFunction - callback signature for treatOntimeintervalSubscriptions
+*/
+typedef void (*OtisTreatFunction)(std::string tenant, BSONObj* bobjP);
```

Comment:
Why don't using references instead pointers? I.e.

```
typedef void (*OtisTreatFunction)(std::string tenant, BSONObj& bobjP);
```
I think this was the common "style" that we are using in general (except in some exceptions) along the code.